lib/levels/rooms_creation.rb

```diff
@@ -8,68 +8,77 @@ def self.create(week_id,user_id)
       week = Week.find(week_id)
       learning_objects = week.learning_objects.all.distinct
 
-      # Vypocet kolko otazok typu evaluator, resp. choice sa ma do miestnosti vybrat
-      sorted_los = Array.new
-      number_of_evaluator_q = week.free_los("EvaluatorQuestion",user_id).count
-      number_of_choice_q = week.free_los("ChoiceQuestion",user_id).count
-
-      if number_of_evaluator_q > Constants::NUMBER_EVALUATOR_LOS_IN_ROOM # Do miestnosti sa priradi stanoveny pocet evaluator question a ak ich tolko nie je, tak pocet kolko ich dany tyzden ma
-        evaluator = Constants::NUMBER_EVALUATOR_LOS_IN_ROOM
-      else
-        evaluator = number_of_evaluator_q
-      end
-
-      if (number_of_choice_q > (Room::NUMBER_LOS_IN_ROOM - evaluator)) # Zvysok sa doplni choice, opat ak ich tolko je, tak sa doplni do poctu miestnosti, inak len tolko, kolko ich tyzden ma
-        choice = Room::NUMBER_LOS_IN_ROOM - evaluator
+      if week.free_los("EvaluatorQuestion",user_id).count == 0
+        nil
```

Comment:
predpokladam, ze tento `nil` sa ma povazovat za navratovu hodnotu metody. Takyto zapis moze byt velmi rizikovy, nakolko akonahle niekto prida nejaky kod za cely if-else-end blok, tak uz metoda `nil` nevrati.

Riesil by som to skor takto:
```
def self.create
  unless week.free_los("EvaluatorQuestion",user_id).count == 0
    ... nejaky super uzasny kod ...

    unless sorted_los.count == 0
      # Vypocet score a vytvorenie miestnosti
      return room.id
    end

  end

  nil
end
```

Mozno este dalsie odporucanie. Odporucam komentovat kod pomocou YARD. Sice sa to momentalne vobec nepouziva v Alefe, ale zacat s tym by sme mohli @mlabaj ? Narazam na navratove hodnoty. Totiz taketo okomentovanie kodu moze byt velmi napomocne.
```
# Metoda vytvori novu miestnost
#
# @param week_id [Integer] cislo tyzdna
# @param user_id [Integer] identifikator pouzivatela
# @return [Integer] vracia ID miestnosti, ak (KEDY TO VRACIA), inak vrati prazdny identifikator (nil)
def self.create(week_id,user_id)
  ...
end
```

YARD: http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md